tests/test_greedy_follower.py

```diff
@@ -46,7 +47,8 @@ def pbar():
 
 
 @pytest.mark.parametrize("test_navmesh", test_navmeshes)
-def test_greedy_follower(test_navmesh, scene_graph, pbar):
+@pytest.mark.parametrize("recompute_navmesh", [True, False])
```

Comment:
Two requests:  
1) Can you lave this test unchanged? I have been using it if I need to make changes to the greedy follower to test it on all ~600 navmeshes, so this test needs to be really fast.

2) For the navmesh test, I think a possible good test would be as follows
-- Compute the (geodesic) shortest path between some pairs of points
-- Recompute the navmesh with the base setting, recompute paths, they should match
-- Recompute navmesh on very different settings (probably half the agent radius?), some, if not all, paths should be different.

Reply:
Thanks for the suggestion. 👍 Done.